vocode/streaming/synthesizer/eleven_labs_synthesizer.py

```diff
@@ -51,17 +52,22 @@ def create_speech(
                 "similarity_boost": self.similarity_boost,
             }
 
+
         response = requests.post(url, headers=headers, json=body, timeout=5)
         if not response.ok:
             raise ValueError(
                 f"Eleven Labs API error: {response.status_code} - {response.text}"
             )
 
+        if response.status_code != 200:
+            print(f'WARNING: Eleven Labs responsed with non-200 response: {response.status_code}')
```

Comment:
we already have a `response.ok` check above!

Reply:
@ajar98 Done!